### ISSUE TITLE
[Update] Non-Existing Website under Open Source Websites Category

### ISSUE THREAD

**Issue by surya-mu** [sdmg15/Best-websites-a-programmer-should-visit#1804]
![image](https://github.com/sdmg15/Best-websites-a-programmer-should-visit/assets/101094019/2b84e8b2-ed8a-4839-8b6e-a3af4d5a7d79)

` The website no longer exists, either the name has been changed, or does not exist anymore.`

**Comment by surya-mu** [comment]
I would like to remove it and submit a PR.

**Comment by stmapman7** [comment]
@surya-mu: Yes, I confirm that this link no longer exists. I can submit a PR to remove the link. 
![image](https://github.com/sdmg15/Best-websites-a-programmer-should-visit/assets/4437127/b0bb7944-da8c-48fb-92a6-874df4fa68fd)
